src/services/AffectService.ts

```diff
@@ -35,3 +35,49 @@ export async function deleteAffect(uuid: string) {
     .then(createSuccessHandler({ title: 'Success!', body: 'Affect Deleted.' }))
     .catch(createCatchHandler('Error deleting Affect:'));
 }
+
+
+
+// {
+// "comment": "string",
+// "cvss_version": "V2",
+// "issuer": "RH",
+// "vector": "string",
+// "embargoed": true
+//   "updated_dt": "2024-02-06T16:02:54.708Z"
+// }
```

Comment:
What is the purpose for this commented objects?
(same for lines 66 - 73)

Reply:
It's there to document the type used in the request, however, declaring the type here like we do in other such `*Service.ts` files may be preferable. This relates to a larger discussion on Zod/TS/OpenAPI type management